crypt/crypt.go

```diff
@@ -83,7 +85,10 @@ const (
 // the algorithm is not selectable/customizable.
 func Crypt(password string) (string, error) {
 	cPassword := C.CString(password)
-	defer C.free(unsafe.Pointer(cPassword))
+	defer func() {
+		C.memset(unsafe.Pointer(cPassword), 0, len(password))
```

Comment:
According to error messages on travis build you can't use len(password) as a parameter.